sopel/bot.py

```diff
@@ -376,6 +376,11 @@ def reply(self, text, dest, reply_to, notice=False):
         else:
             self.say(text, dest)
 
+    def kick(self, text, channel, nick):
+        """Send an IRC KICK to a user in a channel.
+        """
```

Comment:
This closing `"""` should go at the end of the previous line (single-line docstrings don't need line-breaks).